### ISSUE TITLE
List maintainer looking for help

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/awesome#927]
Post here if you run an Awesome list and need help to maintain it or want to hand it over to someone else. Awesome lists thrive on being well-maintained. If you're not able to do so, consider adding more people.

**Comment by TomasVotruba** [comment]
Hi, I used to run Awesome Doctrine, but not last year really. Help would be great
https://github.com/TomasVotruba/awesome-doctrine

**Comment by sindresorhus** [comment]
@TomasVotruba Can you do a post about looking for maintenance help in the [Doctrine mailing list](https://groups.google.com/forum/#!forum/doctrine-user)? That's a good way to find potential collaborators.

**Comment by TomasVotruba** [comment]
@sindresorhus I'm [not so motivated to do so](https://www.tomasvotruba.cz/blog/2017/03/27/why-is-doctrine-dying/).

**Comment by sindresorhus** [comment]
@TomasVotruba That post seems like a good reason to hand it over to someone else though. Others might find value in Doctrine, even if you don't. On another hand, if it's dying, is really worth keeping it in the Awesome list?

**Comment by TomasVotruba** [comment]
> On another hand, if it's dying, is really worth keeping it in the Awesome list?

@sindresorhus I don't think so. Moreover, half of the information is outdated. And the rest promotes over engineered approach to problems.

**Comment by sindresorhus** [comment]
@TomasVotruba Alright, removed. I would add a note in your list that it's deprecated and link to your blog post.

**Comment by lukasz-madon** [comment]
>@TomasVotruba Can you do a post about looking for maintenance help in the Doctrine mailing list? That's a good way to find potential collaborators.

Or you can do what I did. I picked top 2 users from `contributors` tab.

**Comment by TomasVotruba** [comment]
@lukasz-madon Good idea, thanks. I [gave it a try](https://github.com/TomasVotruba/awesome-doctrine/issues/5).

@sindresorhus You have a point. I'll do so if none will appear to take over.

**Comment by aleksandar-todorovic** [comment]
I'm not giving up on [awesome-c](https://github.com/aleksandar-todorovic/awesome-c), but I would appreciate a maintainer or two to help me out in curating it.

**Comment by TomasVotruba** [comment]
@aleksandar-todorovic Try something [like this](https://github.com/TomasVotruba/awesome-doctrine/issues/5)

**Comment by sjfricke** [comment]
@aleksandar-todorovic I would be happy to help with the awesome-c project. I actually have an open pull request on it right now

**Comment by ChristosChristofidis** [comment]
looking for maintainers for awesome-deep-learning

**Comment by RichardLitt** [comment]
Looking for maintainers for: 
- [awesome-conferences](https://github.com/RichardLitt/awesome-conferences) [Issue](https://github.com/RichardLitt/awesome-conferences/issues/106).
- [awesome-styleguides](https://github.com/RichardLitt/awesome-styleguides). [Issue](https://github.com/RichardLitt/awesome-styleguides/issues/37).

**Comment by rajikaimal** [comment]
Hi all :smiley_cat: ! I'm looking for contributors to expand [awesome-express](https://github.com/rajikaimal/awesome-express)

Thank you ! :tada: :

**Comment by grabear** [comment]
I'm currently working on this [awesome-rshiny](https://github.com/grabear/awesome-rshiny) list with a few contributors.  I just started yesterday and I stumbled upon this thread today.  Any help would be great.  Or if someone could point me in the right direction.  
I think it's pretty well laid out right now.  It just needs some formatting and minor edits and descriptions.  But I know I'm probably missing a few things.
